docs/components/Typography/TextStylesPage.tsx

```diff
@@ -0,0 +1,176 @@
+import { Grid } from '@mui/material';
+import Table from '@mui/material/Table';
+import TableBody from '@mui/material/TableBody';
+import TableCell from '@mui/material/TableCell';
+import TableContainer from '@mui/material/TableContainer';
+import TableHead from '@mui/material/TableHead';
+import TableRow from '@mui/material/TableRow';
+
+import { Typography } from '../../../src/index';
+import TypographyTop from '../Typography/typography-top';
+
+function createData(
+  category: string,
+  styleName: string,
+  fontStyles: string,
+  alignment: string,
+  size: string,
+  capitalization: string,
+) {
+  return { category, styleName, fontStyles, alignment, size, capitalization };
+}
+
+const rows = [
+  createData('H1', 'Header Large', 'Medium', 'Left', '24px', 'Title'),
+  createData('H2', 'Header Small', 'Medium', 'Left', '20px', 'Title'),
+  createData('H3', 'Title Large', 'Regular', 'Left', '32px', 'Title'),
+  createData('H4', 'Title Small', 'Regular', 'Left', '16px', 'Title'),
+  createData('Subheader1', 'Subheader Large', 'Medium', 'Left', '16px', 'Title'),
+  createData('Subheader2', 'Subheader Medium', 'Medium', 'Left', '13px', 'Title'),
+  createData('Subheader3', 'Subheader Small', 'Medium', 'Left', '12px', 'Title'),
+  createData('Subheader4', 'Subheader XSmall', 'Medium', 'Left', '11px', 'Title'),
+  createData('Body1', 'Body Large', 'Regular', 'Left', '16px', 'Sentence'),
+  createData('Body2', 'Body Small', 'Regular', 'Left', '12px', 'Sentence'),
+  createData('Caption', 'Caption', 'Light Italic', 'Left', '11px', 'Sentence'),
+];
+
+const tableCellStyles = {
+  width: 122,
+  height: 40,
+  padding: 0,
+  color: '#ffffff',
+  borderColor: '#6b6b6b',
+};
+const TextStylesPage = () => {
```

Comment:
add the type for this function like `const TextStylesPage: React.FC = () => {...}`